src/program.js

```diff
@@ -78,10 +80,11 @@ export class Program {
     return this;
   }
 
-  async execute(absolutePackageDir: string,
-      {systemProcess = process, logStream = defaultLogStream,
-       getVersion = defaultVersionGetter, shouldExitProgram = true}
-      : Object = {}): Promise<void> {
+  async execute(
+    absolutePackageDir: string,
+    {systemProcess = process, logStream = defaultLogStream,
```

Comment:
Nit, imho this kind of function declaration can be more readable written as follow (but as a matter of taste, it is not a required change for this pull request, just an opinion):

```js
async execute(
  absolutePackageDir: string,
  {
    systemProcess = process, logStream = defaultLogStream,
    ...
  }: Object = {}
): Promise<void> {
  ...
}
```